src/modules/device/README.md

```diff
@@ -18,15 +18,78 @@ They are accessible by, e.g::
 ```
 this.$device.isMobile
 ```
-Or in asyncData by (you need to import Vue):
-```
-Vue.prototype.$device.isMobile
-```
+If you want to use it server-side then just use `serverPrefetch` from Vue 2.6
 
-We could totally disable this feature by setting `config.device.appendToInstance` to false and the small library will not by imported.
+Example config:
+```js
+"device": {
+    "appendToInstance": true,
+    "tests": [
+      "isMobile",
+      "isMobileOrTablet",
+      "isTablet",
+      "isDesktop",
+      "isDesktopOrTablet",
+      "isIos",
+      "isWindows",
+      "isMacOS"
+    ]
+  }
+  ```
+
+`appendToInstance` decides whether module will inject `$device` to Vue instances or not.
 
 In addition when we are using installer script. I've added multiselect so we could pick which tests we want to have.
 
 I've tested it with `curl -A "some user agent" http://localhost:3000 and it worked.
+I prepared `test.js` testing script. If you want to make tests in your website, please uncomment `router.addRoutes` part in module's index.ts and register module in `src/modules/client.ts` - it will add route with values of tests. You also should enable every test. Then just run:
+```sh
+node test.js
+```
+And it will send requests to `http://localhost:3000/device-mod-test` with different user agents. You could multiply amount of tests with `multiplier` variable in test.js file. Ten is a good default value in my opinion.
+
+Anyway we should use media queries whenever we can. However, sometimes we have more advanced structure and we are ending with hidden useless vue instance.
+
+## How to install
+1. Setup config
+2. Import module in `src/modules/client.ts` just like (for using test.js script you need to register module with `registerModule` method):
+```js
+import './device/index';
```

Comment:
there is no need to do this step, because you need to import `import { injectDeviceTests } from 'src/modules/device';` So with that import `once` will be called